atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -1204,6 +1218,7 @@ private static MultiTableSweepQueueWriter uninitializedTargetedSweeper(
         TimestampManagementService timestampManagement();
         TimelockService timelock();
         Optional<TimeLockMigrator> migrator();
+        Optional<LockWatchManager> lockWatcher();
```

Comment:
why have this be optional if you have a noop version?

Reply:
I think when this was originally written we needed an rpc client for the noop impl, but given that we have timelock here, we can just derive what we need when get commit timestamp gets added